admin/load.php

```diff
@@ -353,6 +353,11 @@ function perflab_admin_pointer( $hook_suffix ) {
 		return;
 	}
 
+	// Do not show admin pointer in multisite Network Admin.
+	if ( is_network_admin() ) {
```

Comment:
While that's a separate problem, we should also ensure this pointer doesn't show in the User Admin. That's yet another separate admin area that exists in WordPress. I mentioned that (though a bit as a side note) on the issue description for #387.
```suggestion
	if ( is_network_admin() || is_user_admin() ) {
```

Reply:
Thanks, @felixarntz for the feedback. I missed that mentioned comment.

Suggested changes are updated in PR.

Reply:
@mukeshpanchal27 Excellent, thanks!